src/libs/ReportUtils.ts

```diff
@@ -1872,55 +1886,71 @@ function canEditMoneyRequest(reportAction: OnyxEntry<ReportAction>, fieldToEdit
     }
 
     const moneyRequestReport = getReport(String(moneyRequestReportID));
-    const isReportSettled = isSettled(moneyRequestReport?.reportID);
-    const isApproved = isReportApproved(moneyRequestReport);
-    const isAdmin = isExpenseReport(moneyRequestReport) && (getPolicy(moneyRequestReport?.policyID ?? '')?.role ?? '') === CONST.POLICY.ROLE.ADMIN;
+    const policy = getPolicy(moneyRequestReport?.policyID ?? '');
+    const isAdmin = isExpenseReport(moneyRequestReport) && policy.role === CONST.POLICY.ROLE.ADMIN;
+    const isManager = isExpenseReport(moneyRequestReport) && currentUserAccountID === moneyRequestReport?.managerID;
     const isRequestor = currentUserAccountID === reportAction?.actorAccountID;
-    const isDistanceRequest = !isEmpty(transaction) && TransactionUtils.isDistanceRequest(transaction);
 
-    if (isAdmin && !isRequestor && fieldToEdit === CONST.EDIT_REQUEST_FIELD.RECEIPT) {
-        return false;
-    }
-    if (isDistanceRequest && fieldToEdit === CONST.EDIT_REQUEST_FIELD.AMOUNT) {
-        return isAdmin;
-    }
-    if (isAdmin) {
+    // Admin & managers can always edit coding fields such as tag, category, billable, etc. As long as the report has a state higher than OPEN.
+    if ((isAdmin || isManager) && !isDraftExpenseReport(moneyRequestReport)) {
```

Comment:
I see we need to check here only for the state and status, could we just add it like this since we do not need this logic anywhere else? This can be refactored in future if we would need more things to check for
```suggestion
    if ((isAdmin || isManager) && moneyRequestReport?.stateNum > CONST.REPORT.STATE_NUM.OPEN && moneyRequestReport?.statusNum > CONST.REPORT.STATUS_NUM.OPEN) {
```

Reply:
Looking now into this method, this will allow the manager (the payer) of the IOU report to edit the request even after it have been paid. Since the `isManager` will be true and the `!isDraftExpenseReport(moneyRequestReport)` will also be true since its IOU report and state status is more

Reply:
I think you might be covering those cases in the `canEditFieldOfMoneyRequest ` method but honestly its a bit confusing as the `canEditMoneyRequest` will return true when the user should not be able to edit it in this case

Reply:
>Looking now into this method, this will allow the manager (the payer) of the IOU report to edit the request even after it have been paid. Since the isManager will be true and the !isDraftExpenseReport(moneyRequestReport) will also be true since its IOU report and state status is more

I don't think so, `isManager` can only be `true` if the report is an `expense` report and the user is a manager

`const isManager = isExpenseReport(moneyRequestReport) && currentUserAccountID === moneyRequestReport?.managerID;`